src/main/java/sirius/kernel/cache/Cache.java

```diff
@@ -157,6 +157,8 @@
      */
     Cache<K, V> addRemover(@Nonnull String discriminator, @Nonnull BiPredicate<String, CacheEntry<K, V>> test);
 
+    <T> CacheRemoverBuilder<K, V, CacheEntry<K, V>> addRemover(@Nonnull String discriminator);
```

Comment:
the T can be killed (?) - it is unused..

Reply:
ups. Ja.... noch nicht aufgeräumt und so^^